.github/workflows/haskell.yml

```diff
@@ -94,6 +94,15 @@ jobs:
             cabal_version: 3.6.2.0
             cabal_project: cabal.project
             ignore_error: false
+          - name: 9.4.2-fusion-inspection-Werror
+            ghc_version: 9.4.2
+            runner: ubuntu-latest
+            build: cabal
+            cabal_version: 3.8.1.0
+            disable_sdist_build: "y"
+            cabal_project: cabal.project
```

Comment:
The test title says Werror but it is not using the Werror cabal-project file. Also, we will have to fix the warnings for 9.4.2 to make the Werror CI work.

Reply:
We do not need to fix the warnings in the IsStream/Prelude modules as those are going to be deprecated.